### ISSUE TITLE
Umbrella header 'RecaptchaEnterprise.h' not found

### ISSUE THREAD

**Issue by sebas3407** [GoogleCloudPlatform/recaptcha-enterprise-mobile-sdk#8]
Hi,  I am trying to integrate the library to my application, but when I try to compile it, I am receiving this message: Umbrella header 'RecaptchaEnterprise.h' not found, for the integration I have followed this manual https://cloud.google.com/recaptcha-enterprise/docs/instrument-ios-apps#swift

**Comment by walterjgsp** [comment]
Hi @sebas3407, thanks for reaching us. What have you used to setup your environment? SPM, CocoaPods or direct download?

**Comment by sebas3407** [comment]
Hi @walterjgsp , I have the problem when I use CocoaPods, I have tried using SPM and it works fine for me, however, when initializing the app it gives me an error code 100: _
RecaptchaClient creation error: Error code: 100.

I have searched the documentation, but no I find nothing about the error codes, do you know what that code means?

**Comment by walterjgsp** [comment]
I'm glad that worked with the SPM, I will take a look on the CocoaPods and update the documentation to see what's missing there. The `Error code 100` means that some internal error occur on the SDK. What's the minimum version of your iOS? And what version of Swift and Xcode are you currently using?

**Comment by sebas3407** [comment]
Hello @walterjgsp , I have solved the problem running the app on a physical device, I was able to get the token correctly, in case it helps you, the problem with the iPhone simulator was using Xcode 14.0 and Swift 5

**Comment by mcorner** [comment]
@sebas3407 Did you move back to using Xcode 13 or some other solution?  We are digging into this.

**Comment by sebas3407** [comment]
Hi @mcorner, no, I just ran my app on a physical device and was able to get the token, after that I tried again on the iPhone simulator and got the error code: 100 again, so I guess the problem is with the simulator

By the way, I created a new blank project, I installed the library using Cocapods and I get the error again: Umbrella header 'RecaptchaEnterprise.h' not found ,but with SPM it works perfect, it's very strange

**Comment by walterjgsp** [comment]
Hi @sebas3407, for the *Umbrella header 'RecaptchaEnterprise.h' not found* issue I was able to reproduce the Cocoapods header having both the `import RecaptchEnterprise` on the Swift file and `#import <RecaptchaEnterprise/RecaptchaEnterprise.h>` on my bridging-header file. It started to work for me on Cocoapods when I removed the Swift import. After the removal I was able to build and to launch on an emulator using Cocoapods. 

About the error code 100 on the simulator what's the processor version of your Mac? M1, M2 or Intel?
